includes/Checker/Preparations/Force_Single_Plugin_Preparation.php

```diff
@@ -0,0 +1,97 @@
+<?php
+/**
+ * Class WordPress\Plugin_Check\Checker\Preparations\Force_Single_Plugin_Preparation
+ *
+ * @package plugin-check
+ */
+
+namespace WordPress\Plugin_Check\Checker\Preparations;
+
+use WordPress\Plugin_Check\Checker\Preparation;
+use Exception;
+
+/**
+ * Class for the preparation to force the plugin to be checked as the only active plugin.
+ *
+ * This ensures the plugin is checked as much in isolation as possible.
+ *
+ * @since n.e.x.t
+ */
+class Force_Single_Plugin_Preparation implements Preparation {
+
+	/**
+	 * Plugin slug.
+	 *
+	 * @since n.e.x.t
+	 * @var string
+	 */
+	protected $plugin_basename;
+
+	/**
+	 * Sets the plugin slug.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @param string $plugin_basename Slug of the plugin, E.g. "akismet\akismet.php".
+	 */
+	public function __construct( $plugin_basename ) {
+
```

Comment:
Let's avoid these unnecessary blank lines on top of the method bodies (same applies below).
```suggestion
```